Makefile

```diff
@@ -13,28 +27,14 @@ build: ## build the binary
 install: build ## build and install to /usr/local/bin/
 	cp $(EXECUTABLE_NAME) /usr/local/bin/$(EXECUTABLE_NAME)
 
-test: lint
-	go test ./...
+test: lint ## go test
+	go test ./... -count=1
```

Comment:
☝️ This ensures against false positives running the tests in between code or testdata changes.

Reply:
Note that I also added code comments to all targets within the `Makefile`, such that `make help` documents all targets.